nservicebus/scheduling/index.md

```diff
@@ -51,9 +52,8 @@ snippet: ScheduleTask
  * Since the Scheduler is non-durable, if the process restarts, all scheduled tasks (that are created during the endpoint's startup) are recreated and given new identifiers. Tasks scheduled before the restart that arrive at the endpoint queue will not be found although a message will be written to the log as information.
  * Scheduled tasks, once created, cannot be canceled or changed.
  * While the task repeat interval can be specified, setting a specific time for the task to run is not supported by the Scheduler.
- * Each scheduled task will be executed in a new task generated by the Task Parallel Library using [`Task.Factory.StartNew`](https://msdn.microsoft.com/en-au/library/dd321439.aspx).
-  * There will be no transaction scope by default although one can be created if needed.
-  * If the time for a task is longer than the configured interval then the same action may be executed on different threads concurrently. If an action is not thread safe then the action needs to handle any synchronization logic, for example using a Mutex.
+ * The task will be participate in [the transport transaction](/transports/transactions.md) and is therefor exposed to the transaction timeout.
+ * If the time for a task is longer than the configured interval then the same action may be executed on different threads concurrently. If an action is not thread safe then the action needs to handle any synchronization logic, for example using a Mutex.
```

Comment:
Is mutex deliberate here because it can be used cross process? Isn't semaphore more widely known than mutex? I know it was already there and happy to make it out of scope I was just wondering and figured it doesn't hurt to mention

Reply:
yea, semaphore is better. Changed